src/ServiceWorkerCacher.ts

```diff
@@ -1,15 +1,18 @@
 import Cacher from "./Cacher";
+import Store from "./Store";
 import Manifest from "./Manifest";
 
 /** Class that caches URLs using ServiceWorker's Cache API. */
 export default class ServiceWorkerCacher implements Cacher {
     private serviceWorkerPath: string;
+    private store: Store;
     private supported: boolean;
```

Comment:
This could be given a more descriptive name. Perhaps `areServiceWorkersSupported` is better?